Library/Tracking/Koala.swift

```diff
@@ -1393,6 +1431,51 @@ public final class Koala {
     )
   }
 
+  public func trackCreatorDetailsClicked(project: Project,
```

Comment:
It looks so much better to me when the first arg is on a new line like `self.track` below 😂